contracts/GenArt721CoreV3.sol

```diff
@@ -63,6 +63,9 @@ contract GenArt721CoreV3 is ERC721, IGenArt721CoreContractV3 {
     /// next project ID to be created
     uint256 public nextProjectId = 0;
 
+    /// version of this core contract
+    string public constant coreVersion = "v3.0.0";
```

Comment:
So strong opinion here, but that SGTM personally ya.

Reply:
sure, @yoshiwarab in this case, are you thinking have a public const `name = GenArt721CoreV3`?

Reply:
Yeah was thinking something similar to minterType on IFilteredMinterV0

Reply:
@yoshiwarab pushed an update with a `coreType = "GenArt721CoreV3"`

It seems **slightly** redundant since we wouldn't expect that to change for anything with `coreVersion=^3`, but I think it makes sense to have to allow e.g. minters easily confirm they are working with a GenArt721CoreV3 core.

(also, I don't ever expect there to be a core version v3.0.1, but we should always be prepared 🙂 )

Reply:
This coreVersion/coreType setup LGTM and makes sense :)